app/Filament/Resources/LeaderboardResource.php

```diff
@@ -257,6 +267,11 @@ public static function table(Table $table): Table
                         ->icon('fas-clock-rotate-left'),
                 ]),
             ])
+            ->recordUrl(
+                fn (Leaderboard $record) => $user->can('update', $record)
+                    ? LeaderboardResource::getUrl('edit', ['record' => $record])
+                    : LeaderboardResource::getUrl('view', ['record' => $record])
```

Comment:
The most likely reason to go to the leaderboard management page is to edit the entries, which ties in to my other comment - since that's doable on the details page, should the user be immediately directed to the edit page?